common/beerocks/bcl/source/beerocks_socket_thread.cpp

```diff
@@ -184,6 +184,11 @@ bool socket_thread::handle_cmdu_message_uds(Socket *sd)
         return false;
     }
 
+    // On UDS bus, unsubscribed messages needs to be filtered manually.
```

Comment:
This completely breaks the abstraction! Why didn't you just add this to `btl_uds.cpp` before calling `handle_cmdu`? Why add it to the socket thread class? This whole commit is wrong.

Reply:
Why do you say it is breaking the abstraction? This function is implemented on socket_thread, and can be overridden if needed.
The socket thread is base class for btl classes, and this function is needed for abstraction.